### ISSUE TITLE
Error occurs when removing the last uploaded item of the Upload component

### ISSUE THREAD

**Issue by NightSpark75** [ant-design-blazor/ant-design-blazor#609]
### Describe the bug
Upload組件，移除最後一個上傳的項目會返回錯誤
依然可以將項目移除，但是會返回錯誤

Error occurs when removing the last uploaded item of the Upload component.
The item can be removed but the below exception occurs:

### Exceptions
Microsoft.AspNetCore.Components.WebAssembly.Rendering.WebAssemblyRenderer[100]
      Unhandled exception rendering component: Unknown edit type: 0
      Error: Unknown edit type: 0

### Further technical details
- AntDesign Nuget Package version 0.3.0
- The IDE VS 2019

**Comment by anddrzejb** [comment]
@NightSpark75  can you provide the code? I was not able to recreate it...

**Comment by anddrzejb** [comment]
I missed it because the problem does not exist in Sever side. It seems to be connected to [this issue](https://github.com/dotnet/aspnetcore/issues/26809).

**Comment by ElderJames** [comment]
😂 Can we fix it in our component?

**Comment by anddrzejb** [comment]
Debugging this was a real pain... For some reason none of my browser's javascript debuggers were able to stop on this error. `try...catch` blocks were not working, breakpoint were not stopping, `debugger;` was not working... I finally tracked it down to the `Tooltip` and its `onfocusout` event. Which actually is not `Tooltip` but `Overlay`. I tried:
* catching that it is a `onfocusout` and deferring it as suggested in aforementioned link (not working, regardless of the timeout set)
* disposing of the `Tooltip` before removing the upload item (with all combinations of `StateHasChanged()` and `Task.Delay(100)`
* rolling back to a bound `Tooltip` (other error of type `Event xxx is already tracked`)
* manually removing the `onfocus` event from `Tooltip`'s `OverlayTrigger` (no luck here, the event seems to persist)
Whatever I do, I can't remove the event before it fires... I will try to experiment more, but any ideas are welcome. 

The only thing that worked was totally removing `Tooltip`, but that is not a solution...
